tensorflow/core/kernels/gpuprim.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
```

Comment:
follow the header template from the other header files in TF source. 
upstream will reject otherwise